backend/src/services/release.ts

```diff
@@ -237,9 +239,11 @@ export async function newReleaseComment(user: UserInterface, modelId: string, se
 export async function getModelReleases(
   user: UserInterface,
   modelId: string,
+  q?: string,
 ): Promise<Array<ReleaseDoc & { model: ModelInterface; files: FileInterface[] }>> {
+  const query = q === undefined ? { modelId } : getQuerySyntax(q, modelId)
```

Comment:
This can be simplified to `const query = getQuerySyntax(q, modelId)` as you've already covered off the case of `q === undefined` inside of `getQuerySyntax`

Reply:
Removed the check in getquerysyntax, dealing with it earlier makes more sense to me, but willing to change